node-perf-dash.k8s.io/deployment.yaml

```diff
@@ -8,6 +8,10 @@ metadata:
     version: "0.3"
 spec:
   replicas: 1
+  selector:
+    matchLabels:
+      app: node-perf-dash
+      version: "0.3"
```

Comment:
I don't think you want the version label - it won't match the labels in template below.

Reply:
Fixed.